.buildconfig.yml

```diff
@@ -112,6 +112,10 @@ projects:
     path: components/feature/search
     description: 'Feature implementation connecting an engine implementation with the search module.'
     publish: true
+  feature-searchwidget:
```

Comment:
Why did we want a separate component? Would the `feature-search` have been fine? Just curious. 🙂

Reply:
Done